modules/claims_api/lib/claims_api/vbms_sidekiq.rb

```diff
@@ -6,7 +6,7 @@ module ClaimsApi
   module VBMSSidekiq
     def rescue_file_not_found(power_of_attorney)
       power_of_attorney.update(
-        status: 'failed',
+        status: ClaimsApi::PowerOfAttorney::ERRORED,
```

Comment:
this seemed like a one-off use of `"failed"` as the status value. I refactored this to the `ERRORED` constant to consolidate all POA errors together